content/content-management/front-matter.md

```diff
@@ -149,7 +149,7 @@ If neither `slug` nor `url` is present and [permalinks are not configured otherw
 
 You can add fields to your front matter arbitrarily to meet your needs. These user-defined key-values are placed into a single `.Params` variable for use in your templates.
 
-These fields in the following example can be accessed via `.Params.include_toc` and `.Params.show_comments`, respectively. The [Variables][] section provides more information on using Hugo's page- and site-level variables in your templates.
+The following fields can be accessed via `.Params.include_toc` and `.Params.show_comments`, respectively. The [Variables][] section provides more information on using Hugo's page- and site-level variables in your templates.
 
 ```yaml
```

Comment:
@digitalcraftsman Think we should add both a YAML and TOML example here for consistency? You also mentioned not using snake casing. I personally don't use any casing in my front matter variables because of all the previous conversations surrounding lower vs upper. Thoughts?